### PR TITLE
Updated pinout, pma description, ISA description, exception and inter…

### PR DESCRIPTION
**Pull request by Silabs-ArjanB**
…rupt info

Signed-off-by: Arjan Bink <Arjan.Bink@silabs.com>

### CODE REVIEW

File: docs/user_manual/source/instruction_fetch.rst [hunk 1]
```diff
@@ -27,23 +27,25 @@ are possible and thus it needs fewer signals.
...
-  | ``instr_err_i``         | input           | An instruction interface error occurred                                                                                                           |
-  +-------------------------+-----------------+---------------------------------------------------------------------------------------------------------------------------------------------------+
+  +----------------------------+-----------------+---------------------------------------------------------------------------------------------------------------------------------------------------+
+  | **Signal**                 | **Direction**   | **Description**                                                                                                                                   |
+  +----------------------------+-----------------+---------------------------------------------------------------------------------------------------------------------------------------------------+
+  | ``instr_req_o``            | output          | Request valid, will stay high until ``instr_gnt_i`` is high for one cycle                                                                         |
```
Comment (silabs-oysteink) [inline docs/user_manual/source/instruction_fetch.rst:33]: Should the instr_reqpar_o also be in the table? Same for all other *_par signals

File: docs/user_manual/source/instruction_fetch.rst [hunk 2]
```diff
@@ -27,23 +27,25 @@ are possible and thus it needs fewer signals.
...
+  | ``instr_rvalid_i``         | input           | ``instr_rdata_i`` and ``instr_err_i`` are valid when ``instr_rvalid_i`` is high. This signal will be high for exactly one cycle per request.      |
+  +----------------------------+-----------------+---------------------------------------------------------------------------------------------------------------------------------------------------+
+  | ``instr_rdata_i[31:0]``    | input           | Data read from memory                                                                                                                             |
+  +----------------------------+-----------------+---------------------------------------------------------------------------------------------------------------------------------------------------+
+  | ``instr_err_i``            | input           | An instruction interface error occurred                                                                                                           |
+  +----------------------------+-----------------+---------------------------------------------------------------------------------------------------------------------------------------------------+
```
Comment (silabs-oysteink) [inline docs/user_manual/source/instruction_fetch.rst:49]: Missing *achk / *rchk ?

File: docs/user_manual/source/load_store_unit.rst [hunk 3]
```diff
@@ -12,31 +12,33 @@ supported.
...
+  |                           |                 | transactions. For a read transaction ``data_rdata_i`` holds valid data when ``data_rvalid_i`` is high.                       |
+  +---------------------------+-----------------+------------------------------------------------------------------------------------------------------------------------------+
+  | ``data_rdata_i[31:0]``    | input           | Data read from memory. Only valid when ``data_rvalid_i`` is high.                                                            |
+  +---------------------------+-----------------+------------------------------------------------------------------------------------------------------------------------------+
+  | ``data_err_i``            | input           | A data interface error occurred. Only valid when ``data_rvalid_i`` is high.                                                  |
+  +---------------------------+-----------------+------------------------------------------------------------------------------------------------------------------------------+
```
Comment (silabs-oysteink) [inline docs/user_manual/source/load_store_unit.rst:42]: Same as for instruction interface, missing *par, *achk and *rchk?

**Review by silabs-oysteink — CHANGES_REQUESTED** [review]
We should include all obi signals in the tables describing them (they are included in the instantiation template)

File: docs/user_manual/source/instruction_fetch.rst (hunk 1 above)
Comment (Silabs-ArjanB) [reply docs/user_manual/source/instruction_fetch.rst:33]: I know I left these out (also didn't upload the new OBI spec yet); will add a preliminary description.

File: docs/user_manual/source/instruction_fetch.rst (hunk 2 above)
Comment (Silabs-ArjanB) [reply docs/user_manual/source/instruction_fetch.rst:49]: I know I left these out (also didn't upload the new OBI spec yet); will add a preliminary description.

File: docs/user_manual/source/load_store_unit.rst (hunk 3 above)
Comment (Silabs-ArjanB) [reply docs/user_manual/source/load_store_unit.rst:42]: Will add

**Review by silabs-oysteink — APPROVED** [review]

**Pull request merged**